src/h5_async_vol.c

```diff
@@ -8878,21 +8842,68 @@ async_dataset_read_merge_mdset_col(async_instance_t *aid, size_t count, H5VL_asy
 
 done:
     return found_task;
-} // End async_dataset_read_merge_mdset_col > 1.13.3
+} // End async_dataset_read_merge_mdset_col
```

Comment:
This is not getting displayed correctly in terms of the diff.

Reply:
To get the changes' renderings correct, you will need to use a tool like Meld for the git diff.